From ec45f6dd5fa1d71a3e3c1fc990f686c26e5c8eed Mon Sep 17 00:00:00 2001
From: Nick Krasnoff <krasnoffn@ainfosec.com>
Date: Mon, 10 Aug 2020 15:23:23 -0400
Subject: [PATCH] Fix typo in EFI error codes

There was a long standing typo in the EFI error codes which due to
some external update was exposed. This is already fixed in shim development
so this patch can be removed when upgrading to the next release.

Signed-off-by: Nick Krasnoff <krasnoffn@ainfosec.com>
---
 lib/console.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/console.c b/lib/console.c
index 3aee41c..792aa58 100644
--- a/lib/console.c
+++ b/lib/console.c
@@ -445,7 +445,7 @@ static struct {
 	{  EFI_SECURITY_VIOLATION,     L"Security Violation"},
 
 	// warnings
-	{  EFI_WARN_UNKOWN_GLYPH,      L"Warning Unknown Glyph"},
+	{  EFI_WARN_UNKNOWN_GLYPH,      L"Warning Unknown Glyph"},
 	{  EFI_WARN_DELETE_FAILURE,    L"Warning Delete Failure"},
 	{  EFI_WARN_WRITE_FAILURE,     L"Warning Write Failure"},
 	{  EFI_WARN_BUFFER_TOO_SMALL,  L"Warning Buffer Too Small"},
-- 
2.17.1

